frontends/election-manager/src/components/election_manager_tally_report.tsx

```diff
@@ -1,298 +1,74 @@
-import React, { useContext, useEffect, useRef, useState, useMemo } from 'react';
-import styled from 'styled-components';
-
-import { useParams } from 'react-router-dom';
-import { assert, find, filterTalliesByParams } from '@votingworks/utils';
-import { LogEventId } from '@votingworks/logging';
-import {
-  ExternalTally,
-  VotingMethod,
-  getLabelForVotingMethod,
-  Tally,
-  unsafeParse,
-  PartyIdSchema,
-} from '@votingworks/types';
 import {
```

Comment:
I recommend reviewing this file using [the `?w=1` param](https://github.com/votingworks/vxsuite/pull/1701/commits/47f67e85eed54117e5d47162873a9859c167395b?w=1)